src/v/datalake/translation/partition_translator.cc

```diff
@@ -202,12 +203,15 @@ ss::future<> partition_translator::translate_until_stopped() {
         }
 
         auto last_committed_offset = result.last_iceberg_committed_offset;
-        auto last_translated_offset = result.last_added_offset.value_or(
+        // LTO stands for last translated offset
+        auto lto_from_coordinator = result.last_added_offset.value_or(
           kafka::prev_offset(_data_source->min_offset_for_translation()));
-
+        // TODO: maybe we should store the translation timestamp in
+        // coordinator
+        // ?
```

Comment:
I had added some of this new state to the coordinator before, and it felt out of place since these timestamps are not an absolute quantity with respect to either the catalog or the source ntp. 

if a translator comes up and there's no "last timestamp" there, that doesn't effect the semantics of translation; we will just translate some and generate a timestamp ad hoc.

I don't feel too strongly about it though, just my $0.02.